tests/test_suites/ShortSystemTests/test_read_write_during_scan.sh

```diff
@@ -2,12 +2,21 @@ timeout_set 2 minutes
 
 # Create an installation with 3 chunkservers, 3 disks each.
 # All disks in CS 0 will fail during the test.
-USE_RAMDISK=YES \
-	MOUNTS=1
-	CHUNKSERVERS=3 \
-	DISK_PER_CHUNKSERVER=3 \
-	MOUNT_EXTRA_CONFIG="sfscachemode=NEVER|sfsdirectio=1"
-	setup_local_empty_saunafs info
+if is_windows_system; then
+	USE_RAMDISK=YES \
+		MOUNTS=1 \
+		CHUNKSERVERS=3 \
+		DISK_PER_CHUNKSERVER=3 \
+		MOUNT_EXTRA_CONFIG="sfscachemode=NEVER"
+		setup_local_empty_saunafs info
+else
+	USE_RAMDISK=YES \
+		MOUNTS=1 \
+		CHUNKSERVERS=3 \
+		DISK_PER_CHUNKSERVER=3 \
+		MOUNT_EXTRA_CONFIG="sfscachemode=NEVER|sfsdirectio=1"
+		setup_local_empty_saunafs info
+fi
```

Comment:
Please consider to only modify the involved line. I mean the if should just add the "|sfsdirectio=1" to the MOUNT_EXTRA_CONFIG if it is on Windows.

Reply:
Done